gitlab/__init__.py

```diff
@@ -880,7 +880,30 @@ def getprojectissue(self, project_id, issue_id):
             return request.json()
         else:
             return False
+    def getopenprojectissue(self, project_id):
+        """Get all opened issues from a project
 
+        :param project_id: project id
+        :return: the opened issue
+        """
+        request = requests.get("{0}/{1}/issues?state=opened".format(self.projects_url, project_id),
+                               headers=self.headers, verify=self.verify_ssl, auth=self.auth, timeout=self.timeout)
+        if request.status_code == 200:
+            return request.json()
+        else:
+            return False
+    def getcloseprojectissue(self, project_id):
```

Comment:
Shouldn't this be get_closed_project_issues instead. I know the style here isn't human friendly (which is strange, since this is a human API and not machine).
If not, then perhaps at least getopene_d_projectissue_s_